src/smc-webapp/project_page.cjsx

```diff
@@ -186,7 +184,38 @@ FreeProjectWarning = rclass ({name}) ->
             </ul>
         </div>
 
+    render_learn_more : ->
+        <a onClick={=>@actions(project_id: @props.project_id).show_extra_free_warning()}> learn more...</a>
```

Comment:
Replacing with 
```
onClick={@actions(project_id: @props.project_id).show_extra_free_warning}>
```  
should work I since `show_extra_free_warning` doesn't take any arguments.